src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/add_sub_menu_request.cc

```diff
@@ -85,7 +85,10 @@ void AddSubMenuRequest::Run() {
   }
 
   const int32_t menu_id = received_msg_params[strings::menu_id].asInt();
-  if (app->FindSubMenu(menu_id)) {
+
+  const auto sub_menu = app->FindSubMenu(menu_id);
+
+  if (smart_objects::SmartType_Null == sub_menu.getType()) {
```

Comment:
```suggestion
  if (smart_objects::SmartType_Null != sub_menu.getType()) {
```

Reply:
Also please fix the log message below, it should say that that submenu id already exsits

Reply:
Fixed, also fixed one more fixed place 63cdee2954